### ISSUE TITLE
Document feature request

### ISSUE THREAD

**Issue by ShadowMasterOfGames** [TheGameCreators/GameGuruRepo#5516]
I think there should be documents about swapping out the stock animation with the boost pack in max cc,. 

1) people would learn that they could do more with the max character creator and
2) might sell more of the animations boost pack if people knew that you could build your own custom max character for fighting, shooting, etc... inside of max cc.

The link below is a video of a max character that I customize in cc with some melee animations from the Animations booster pack, which I did not know was possible until Armed and Hammer, and DBird on discord told me about it.

https://discord.com/channels/950796460683829329/1169650821403783209/1236435076909240400

**Comment by DBird-1994** [comment]
Yes I recently discovered through a bit of trial and error that you can actually replace the default animations with animations from the booster pack.   This is a great feature but as ShadowMasterOfGames stated there is no documentation on this functionality.

To piggyback on this request, it would be great to have even more varieties of attack animations available to swap in and out of our CC creations.   I realize this is something an animator would have to make so I'm not asking for any free content.  But it would be welcome as another animation dlc.  I would suggest things like martial arts, ninja attacks, medieval combat, .   Even more varieties of walk/run and death animations.  It's quite satisfying customizing the animations so not every character looks the same, so more variety would be welcome.

**Comment by Necrym59** [comment]
@ShadowMasterOfGames   Write up a rough document of the procedures used/required and add it here and it will be given a due workover by myself and Bloodmoon into a guide and placed into the guides folder for all.

**Comment by ShadowMasterOfGames** [comment]
@Necrym59 this is how DBird had explained it to me on discord. 

Because I think I'm probably the only one to have ever done this from what I've seen here over the past year,  I'll share my knowledge.  Like I said above, if you're tired of seeing the same old melee attacks, you can swap in some new ones from the animation DLC.

In the CC, I created a character named Wesley Snipes, and swapped out Attack_01 and Attack_02 for the high kick and roundhouse kick from the animation dlc.   Just make sure you tick the "Use Default Animations" box so it uses the animations you've assigned in the CC rather than the stock ones.   You can replace ANY animation on any character with ones from the dlc.  And it works straightaway with the stock character behaviors, no modifications needed.

Here is a link to her first video on discord showing it in action:
https://discord.com/channels/950796460683829329/1169650821403783209/1236113137166454874

Below is a 2 minute video tutorial that DBird made and the instructions.   Here I replace the dagger attack animations with fighting animations from the booster pack.   You can see me first loading the animation set I want to customize, then I delete the individual animations I want to swap out.  Then finally load and rename the new animations, just make sure to check the tickbox "Use Default Animations" in the behavior so it uses the customs anims you've added.   You can replace as many as you want.   I like to replace the stock idle animations with ones from the booster pack, so that all my enemies aren't standing around in the exact same idle pose when they are not attacking me.

https://discord.com/channels/950796460683829329/1169650821403783209/1236391107454046209

sorry the videos were to big load on here, @Necrym59

**Comment by Kitakazi** [comment]
This is just adding animations to a character lol. You are most certainly not the first one to remove animations and replace them with other animations. That is the whole point of the animation booster pack, to add additional animations to a character, or if you so choose replace existing ones.  You can replace any animation from any character as long as they have the same rig. 

I dont know if there is a guide or not on importing animated characters but that's basically what you are suggesting.

**Comment by ShadowMasterOfGames** [comment]
@Kitakazi, necrym59 is going to be making one, if you knew how to do, it would have been helpful for us who did not know. And I agree with DBird, there need to animation booster packs so you can make your own custom characters in max with unique animations.

Thank you so much for your help on this @Necrym59

**Comment by Kitakazi** [comment]
Message me on discord I can try and help you.
